include/seastar/core/disk_params.hh

```diff
@@ -92,6 +93,19 @@ public:
     const std::vector<dev_t>& queue_devices(unsigned q) const {
         return _disks.at(q).devices;
     }
+
+    // Look up physical_block_size override for a device
+    // Returns std::nullopt if no override is configured
+    std::optional<uint32_t> get_physical_block_size(dev_t device) const {
```

Comment:
Presumably this helper is not needed any longer

Reply:
yup, thanks for pointing this out. dropped.